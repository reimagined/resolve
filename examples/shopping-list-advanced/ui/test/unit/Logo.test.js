import React from 'react'
import Logo from '../../src/Logo'

import { shallow } from 'enzyme'

it('renders correctly', () => {
  expect(shallow(<Logo />)).toMatchSnapshot()
})
