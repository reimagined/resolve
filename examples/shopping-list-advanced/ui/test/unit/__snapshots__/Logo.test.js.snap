// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Logo />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <NavbarBrand>
          <ContextBasedComponent
            className="example-icon"
            src="/resolve-logo.png"
          />
          Shopping List
        </NavbarBrand>,
        <NavbarToggle />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <ContextBasedComponent
              className="example-icon"
              src="/resolve-logo.png"
            />,
            "Shopping List",
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "className": "example-icon",
              "src": "/resolve-logo.png",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "Shopping List",
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <NavbarBrand>
            <ContextBasedComponent
              className="example-icon"
              src="/resolve-logo.png"
            />
            Shopping List
          </NavbarBrand>,
          <NavbarToggle />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <ContextBasedComponent
                className="example-icon"
                src="/resolve-logo.png"
              />,
              "Shopping List",
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "className": "example-icon",
                "src": "/resolve-logo.png",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "Shopping List",
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
